LayoutTests/platform/glib/TestExpectations

```diff
@@ -1248,6 +1248,8 @@ webkit.org/b/227122 imported/w3c/web-platform-tests/mathml/relations/css-styling
 webkit.org/b/203146 fast/canvas/offscreen-enabled.html [ Pass ]
 webkit.org/b/203146 http/wpt/offscreen-canvas [ Pass ]
 webkit.org/b/203146 imported/w3c/web-platform-tests/html/canvas/offscreen [ Pass ]
+# After 263731@main:
+webkit.org/b/257969 fast/canvas/offscreen-clipped.html [ ImageOnlyFailure ]
```

Comment:
This use of the prefix bug number is not correct. The bug number should be a bug to track fixing of the test itself, not the landing of the change. I see that this bug, 257969 is used as prefix in all the expectations but it's closed by landing this patch.

Please also note that bugs used to track test failures should only group those that are directly related, perhaps fixable by the same code changes. These bugs should only be closed when the failure can be removed from the respective expectations file.